src/main/resources/application-integrasjonstest.yaml

```diff
@@ -105,6 +105,13 @@ no.nav.security.jwt:
           client-secret: ${AZURE_APP_CLIENT_SECRET}
           client-auth-method: client_secret_basic
 
+tilgang:
+  kode6:
```

Comment:
burde man begynne å bruke fortrolig og strengtFortrolig i stedet for kode6/7?

Reply:
Hmm ja, burde vel kanskje det samtidig som jeg føler vi oftest bruker begrepene kode 6/7 👀